Makefile

```diff
@@ -58,32 +56,27 @@ copycomp:
 
 # Build api docs
 updateapispec:
-	cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/generators/openapi-spec/swagger.json
+	cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/config/swagger.json
 
 api: cleanapi
-	go run gen-apidocs/main.go --config-dir=gen-apidocs/generators --munge-groups=false
+	go run gen-apidocs/main.go --config-dir=gen-apidocs --munge-groups=false
 
-# NOTE: The following "sudo" may go away when we remove docker based api doc generator
 cleanapi:
-	sudo rm -rf $(shell pwd)/gen-apidocs/generators/build
-	sudo rm -rf $(shell pwd)/gen-apidocs/generators/includes
+	rm -rf $(shell pwd)/gen-apidocs/build
 
 copyapi:
+	mkdir -p $(APIDST)
 	cp $(APISRC)/build/index.html $(APIDST)/index.html
-	cp $(APISRC)/build/navData.js $(APIDST)/navData.js
-	cp $(APISRC)/static/scroll.js $(APIDST)/scroll.js
+	# copy scroll.js, jquery.scrollTo.min.js and the new navData.js
+	mkdir -p $(APIDST)/js
+	cp $(APISRC)/build/navData.js $(APIDST)/js/
```

Comment:
Are you creating a new js sub-directory and reference in the index footer?
The current structure is, for example:
$HOME/website/static/docs/reference/generated/kubernetes-api/v1.17/jquery.scrollTo.min.js
$HOME/website/static/docs/reference/generated/kubernetes-api/v1.17/index.html
$HOME/website/static/docs/reference/generated/kubernetes-api/v1.17/scroll.js
$HOME/website/static/docs/reference/generated/kubernetes-api/v1.17/navData.js

Reply:
OK. I see you have changed the k8s/website directory structure for the js files in `index.html`.